config/Config-build.in

```diff
@@ -58,6 +58,10 @@ menu "Global build settings"
 		bool "Enable signature checking in opkg"
 		default SIGNED_PACKAGES
 
+	config CERTIFICATE_CHECK
+		bool "Enable TLS certificate verification during package download"
+		default n
```

Comment:
Should be enabled by default as discussed in this PR

Reply:
Is that the consensus? I've seen both opinions voiced.

Reply:
I am afraid that the option to enable it in the future will take some time or somebody will forget to turn it on. Besides that, who will turn this on by default manually? Would he/she know that there is a new option that can be used? From my side, it is better to have this on in the master branch, so we can catch those failures and fix them. Rather be safe and secure than regret, and if someone regrets, he/she can disable it.